rpc/src/v1/helpers/dispatch.rs

```diff
@@ -396,12 +413,23 @@ impl Dispatcher for LightDispatcher {
 		}))
 	}
 
-	fn sign(&self, accounts: Arc<AccountProvider>, filled: FilledTransactionRequest, password: SignWith)
-		-> BoxFuture<WithToken<SignedTransaction>>
+	fn sign<P>(
```

Comment:
missing space

```suggestion
	fn sign<P> (
```

Reply:
this isn't used consistently across the codebase

Reply:
nvm, I was wrong I think your style is correct sorry!